### DISCUSSION TITLE
TuringCourses/major_basic/convex_optimization/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#146] [General]
# TuringCourses/major_basic/convex_optimization/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_basic/convex_optimization/

**Comment by dieKarotte** [comment]
这门课程比较推荐在大二上还有点数理基础的时候完成修读，不然容易让助教gg血压上升，不过跨校区的上课是一个小小的问题，我分出了周五的时间用于该课程，中午偶尔也可以和泉的学长淦个饭，提前享受玉泉退休生活。23-24秋冬学期有qh和zy老师开班，本人上的是qh老师的班，由于qh老师对到课的宽宥，在学期中只有3人到课，但qh老师还是很有激情的完成了对ppt的read，今年qh的博士生张超老师也来参与了授课，相比qh有稍重的口音但是也会生动地画图解释一些不太好理解的部分。最后由于到课人数实在惨淡，qh老师不得已加入了10分的平时分。对比qh和zy老师的两个班，qh老师班的作业比较偏向于理论的证明，更具有数学风味，而zy老师班的作业对于代码有一定的要求，两者的侧重点不尽相同，但请同学们不要使用chat-gpt完成作业，它的证明真的很不可靠，像是小学生的伪证。本学期没有mid-term的考察，期末的大作业也仅仅是论文的研读与综述撰写，并且延期到了考试周后的寒假。分数上qh给予了极大的宽宥，也谢谢老师和助教。

**Comment by GooduckZ** [comment]
本人上的是 qh 老师的班，在平时的作业中如果遇到不会做的题建议是多看看课件、参考书籍以及金鱼马学长的笔记。直接在网上搜索一般不会有结果，搜到的大概率是你的同学在问答网站上提的问（×）。请不要使用生成式人工智能完成作业，认真负责的助教哥哥会很无奈（）。今年的期末报告给出了十篇论文，按学号指定了你可以选择的其中两篇论文，且选择相同论文的同学可以组队（最多三人）。这样的形式也进一步缓解了大家的压力（）。也谢谢老师和助教。

**Comment by 626lt** [comment]
和楼上一样是大二上修习的这门课，尽管是一门在玉泉的课，但是开课时间是在周五的3，4节，到课的可行性还是挺大的，虽然我最后一次也没去（x）。这门课对不同老师而言可以说是两门课，qh老师的课完全就是一门数学课，对多元函数以及线性代数相关的知识有一定的要求。在zy老师班上据说是有一些代码内容要完成的。qh老师是很nice的，尽管由于他的宽恕，23秋冬上课到场人数基本稳定在3人，这也直接导致了老师在最后评分的阶段不得不加了10分的课堂表现分并且对大部分没到场上课的同学进行了惩罚。所以能够去到现场听课还是很重要的。这门课也是一门相对较为轻松的课，5次的平时作业（老师最后都延期了很久很久）加上一次期末的论文阅读报告（可以组队在寒假完成）就是全部的内容，最后的论文阅读可以组队，是抱紧大腿的好机会（x）。总体而言这门课的workload没那么大，十分建议在大二上完成修习。23秋冬课程的助教哥哥对于平时分的给分也是相当宽裕的，只要认真完成，不要使用生成式人工智能等欺骗行为来应付，助教哥哥会耐心地批改并且指出问题。如果遇到实在不会做的题目，建议参考金鱼马学长在知乎上的笔记以及相应参考书的证明。切记不要使用诸如Chat-gpt等工具进行回答，这对于一个认真批改作业并且仔细阅读你提交的答案的人是非常不负责的行为。
